eng/ignore-links.txt

```diff
@@ -7,3 +7,4 @@ https://docs.microsoft.com/javascript/api/@azure/storage-blob-changefeed
 https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob-changefeed/samples/javascript
 https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob-changefeed/samples/typescript
 https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob-changefeed/test/
+https://github.com/Azure/azure-digital-twins/blob/private-preview/Documentation/how-to-manage-routes.md
```

Comment:
Can we add a section with a comment header that says "# Ignore links to protected sites that we want to have in our public repo".